samples/sampler_object.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<!-- Ported from the OpenGL Samples Pack https://github.com/g-truc/ogl-samples/blob/master/tests/gl-330-sampler-object.cpp -->
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - sampler_object</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="utility.js"></script>
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - sampler_object</div>
+
+    <script id="vs" type="x-shader/x-vertex">
+        #version 300 es
+        #define POSITION_LOCATION 0
+        #define TEXCOORD_LOCATION 4
+
+        precision highp float;
+        precision highp int;
+
+        uniform mat4 mvp;
+
+        layout(location = POSITION_LOCATION) in vec2 position;
+        layout(location = TEXCOORD_LOCATION) in vec2 textureCoordinates;
+
+        out vec2 st;
+
+        void main()
+        {
+            st = textureCoordinates;
+            gl_Position = mvp * vec4(position, 0.0, 1.0) ;
+        }
+    </script>
+
+    <script id="fs" type="x-shader/x-fragment">
+        #version 300 es
+        #define FRAG_COLOR_LOCATION 0
+
+        precision highp float;
+        precision highp int;
+        layout(std140, column_major) uniform;
+
+        struct Material
+        {
+            sampler2D diffuse[2];
+        };
+
+        uniform Material material;
+
+        in vec2 st;
+
+        layout(location = FRAG_COLOR_LOCATION) out vec4 color;
+
+        void main()
+        {
+            if (st.y / st.x < 1.0) {
+                color = texture(material.diffuse[0], st);
+            } else {
+                color = texture(material.diffuse[1], st) * .77;
+            }
+        }
+    </script>
+
+    <script>
+    (function() {
+        'use strict';
+
+        var canvas = document.createElement('canvas');
+        canvas.width = Math.min(window.innerWidth, window.innerHeight);
+        canvas.height = canvas.width;
+        document.body.appendChild(canvas);
+
+        var gl = canvas.getContext( 'webgl2', { antialias: false } );
+        if (!gl) {
+            gl = canvas.getContext( 'experimental-webgl2', { antialias: false } );
+        }
+        var isWebGL2 = !!gl;
+        if(!isWebGL2)
+        {
+            document.getElementById('info').innerHTML = 'WebGL 2 is not available.  See <a href="https://www.khronos.org/webgl/wiki/Getting_a_WebGL_Implementation">How to get a WebGL 2 implementation</a>';
+            return;
+        }
+
+        // -- Initialize program
+
+        var program = createProgram(gl, getShaderSource('vs'), getShaderSource('fs'));
+
+        var uniformMvpLocation = gl.getUniformLocation(program,"mvp");
```

Comment:
Use `'`, instead of `"`.  It doesn't matter, but the samples should be consistent.